src/Header/Header.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './Header.scss'
+
+import home from '../images/house-white.png'
+import post from '../images/post-white.png'
+import feed from '../images/user-white.png'
+import awards from '../images/award-white.png'
+import seLogo from '../images/se-logo.png'
+import { Link, Redirect } from 'react-router-dom';
+
+export default function Header() {
+    return (
+        <section>
+            <section className='header-top'>
+                <img src={seLogo} alt='Saturn`d Earth Logo' id='se-logo'/>
+                <h1 className='header-text'>Saturn'd Earth</h1>
+            </section>
+            <section className='header-bottom'>
+              <Link to="/" label="home">
```

Comment:
We should use `NavLink`s!

They'll make it much easier to visually display which page we're on.